code/game/objects/items/devices/radio/headset.dm

```diff
@@ -163,11 +163,13 @@
 	icon_state = "com_headset"
 	keyslot = new /obj/item/device/encryptionkey/heads/cmo
 
+/*CITADEL CHANGE - comments this out to be overridden in modular_citadel

```

Comment:
Ditto

Reply:
I commented the whole encryption key and headset out for the HoP since I was basically changing the entire item.